joomla/joomla-cms
# [4.0] Check on the component if the categories support is available

Pull Request for Issue #20667.

### Summary of Changes
Adds a check function if associations is available for categories of the current component.

### Testing Instructions
- Enable the Language filter plugin and make sure Associations is set to Yes
- Display Articles Categories Manager

### Expected result
The Associations column should display.

### Actual result
No Associations column is displayed.
